library/src/utils/window.util.ts

```diff
@@ -0,0 +1,8 @@
+export function getOnDocumentReadyPromise(): Promise<void> {
+  return new Promise(resolve => {
+    if (document.readyState == 'complete') {
```

Comment:
`Expected '===' and instead saw '=='.eslint[eqeqeq](https://eslint.org/docs/rules/eqeqeq)`
```suggestion
    if (document.readyState === 'complete') {
```